src/pages/components/data-table/usage.mdx

```diff
@@ -277,148 +280,312 @@ import {
   </ComponentVariant>
 </ComponentDemo>
 
-## Basic data table
+## Overview
 
-The data table component is designed in a way for additional functionality to be added on to fit your team's needs.
+The data table's features are ideal for organizing and displaying data in a UI. The column headers can sort data in ascending or descending order, rows can be expanded to progressively disclose information, and single or batch actions can be taken on rows.
 
-The basic data table is shipped with a base style, which includes:
+The data table toolbar gives a location for primary buttons, search, filtering, table display settings, and other utilities.
 
-- Borders
-- No zebra striping, row dividers instead.
-- No Pagination, search, table toolbar, or multiselect
+### When to use
 
-<div className="image--fixed">
+- To organize and display data.
+- If your user must navigate to a specific piece of data to complete a task.
+- Displaying all of a user's resources.
 
-![basic data table](images/data-table-usage-1.png)
+### When not to use
 
-</div>
+- When a more complex display of the data or interactions are required.
+- As a replacement for a spreadsheet application.
 
-<Caption>Basic data table</Caption>
+### Variations
 
-## Additional functionality
+| Type             | Purpose                                                                                                                                                                                                                             |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Basic_          | The basic data table comes with a base style with only the title, header, and table elements. rows.                                                                                                                                 |
+| _Zebra_          | The zebra data table features alternating colored rows to make scanning horizontal information easier for the user.                                                                                                                 |
```

Comment:
Move zebra stripes to "modifier" section since its a style change and not a UX feature.